command_line_assistant/config.py

```diff
@@ -33,9 +33,36 @@
 [backend]
 endpoint = "{endpoint}"
 verify_ssl = {verify_ssl}
+
+[logging]
+type = "{logging_type}"
 """
 
 
+class LoggingSchema(namedtuple("Logging", ["type", "file"])):
+    """This class represents the [logging] section of our config.toml file."""
+
+    # Locking down against extra fields at runtime
+    __slots__ = ()
+
+    _logging_types: list[str] = ["verbose", "minimal"]
```

Comment:
Do we want this class var to be mutable? A tuple may be a better choice.

Reply:
```suggestion
    _logging_types: list[str] = ("verbose", "minimal",)
```